spec/schema.json

```diff
@@ -412,7 +413,8 @@
         "trust_ids": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "string",
```

Comment:
Minor, but suggest making one of these examples what the value looks like when it's an array of ids. Is that comma-separated? Not sure...

Reply:
I'm not suggesting changing the spec you've defined in any way, just one of the the examples. If it's an array of ids that come in, is that in a comma separated string? I might be missing the point here, in which case, let's have a quick call to dicuss.

Reply:
Ok discussed this and understand the issue (when testing) so happy it stays as is.